src/Symfony/Component/ObjectMapper/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "symfony/object-mapper",
+    "type": "library",
+    "description": "Object to object mapper",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "psr/container": "^2.0"
+    },
+    "require-dev": {
+        "symfony/property-access": "^7.2"
```

Comment:
I think we need a conflict line as well

Reply:
could you be more specific ?

Reply:
I meant, adding `"symfony/property-access": "<7.2"` in the `"conflict"` section to prevent this component to be installed with an old version of the property-access component.